assessment.py

```diff
@@ -177,10 +215,16 @@ def GET(self, bank_id):
     def PUT(self, bank_id):
         try:
             am = autils.get_assessment_manager()
+            data = self.data()
```

Comment:
This is just so I can call the passed in parameters (`data`) more easily in the future without invoking the `self.data()` method.